packages/samples/mvp-dapp/tsconfig.json

```diff
@@ -10,5 +10,6 @@
     "strict": true,
     "skipLibCheck": true
   },
-  "include": ["src"]
+  "include": ["src"],
+  "files": ["src/type-extends.d.ts"]
```

Comment:
This can be fixed by https://github.com/ckb-js/kuai/pull/173. https://github.com/ckb-js/kuai/blob/a8579bf4bd13605f44d2548bbbd5f02f1fd04105/packages/samples/mvp-dapp/tsconfig.json#L13-L15
It should add config for ts-node

Reply:
This config has been added on the develop branch and could be resolved by rebase.

Ref: https://github.com/ckb-js/kuai/blob/develop/packages/samples/mvp-dapp/tsconfig.json#L14

Reply:
> This can be fixed by #173.
> 
> https://github.com/ckb-js/kuai/blob/a8579bf4bd13605f44d2548bbbd5f02f1fd04105/packages/samples/mvp-dapp/tsconfig.json#L13-L15
> 
> 
> It should add config for ts-node

Okay here I'll remove it first